src/routes/collectibles/__tests__/controllers/zerion-collectibles.controller.spec.ts

```diff
@@ -0,0 +1,456 @@
+    })
+      .overrideModule(AccountDataSourceModule)
+      .useModule(TestAccountDataSourceModule)
+      .overrideModule(CacheModule)
+      .useModule(TestCacheModule)
+      .overrideModule(RequestScopedLoggingModule)
+      .useModule(TestLoggingModule)
+      .overrideModule(NetworkModule)
+      .useModule(TestNetworkModule)
+      .compile();
+
+    const configurationService = moduleFixture.get(IConfigurationService);
+    zerionBaseUri = configurationService.get(
+      'balances.providers.zerion.baseUri',
+    );
+    zerionChainIds = configurationService.get(
+      'features.zerionBalancesChainIds',
+    );
+    networkService = moduleFixture.get(NetworkService);
+
+    app = await new TestAppProvider().provide(moduleFixture);
+    await app.init();
+  });
+
+  afterAll(async () => {
+    await app.close();
+  });
+
+  describe('Collectibles provider: Zerion', () => {
+    describe('GET /v2/collectibles', () => {
+      it('successfully gets collectibles from Zerion', async () => {
+        const chain = chainBuilder().with('chainId', zerionChainIds[0]).build();
+        const safeAddress = faker.finance.ethereumAddress();
+        const aTokenAddress = faker.finance.ethereumAddress();
+        const aNFTName = faker.string.sample();
+        const anUrl = faker.internet.url({ appendSlash: true });
+        const zerionApiCollectiblesResponse = zerionCollectiblesBuilder()
+          .with('data', [
+            zerionCollectibleBuilder()
+              .with(
+                'attributes',
+                zerionCollectibleAttributesBuilder()
+                  .with(
+                    'nft_info',
+                    zerionNFTInfoBuilder()
+                      .with('contract_address', aTokenAddress)
+                      .build(),
+                  )
+                  .build(),
+              )
+              .build(),
+            zerionCollectibleBuilder()
+              .with(
+                'attributes',
+                zerionCollectibleAttributesBuilder()
+                  .with(
+                    'nft_info',
+                    zerionNFTInfoBuilder().with('name', aNFTName).build(),
+                  )
+                  .build(),
+              )
+              .build(),
+            zerionCollectibleBuilder()
+              .with(
+                'attributes',
+                zerionCollectibleAttributesBuilder()
+                  .with(
+                    'nft_info',
+                    zerionNFTInfoBuilder()
+                      .with('content', {
+                        preview: { url: anUrl },
+                        detail: { url: anUrl },
+                      })
+                      .build(),
+                  )
+                  .build(),
+              )
+              .build(),
+          ])
+          .build();
+        const chainName = app
+          .get(IConfigurationService)
+          .getOrThrow(
+            `balances.providers.zerion.chains.${chain.chainId}.chainName`,
```

Comment:
Unrelated to this PR but do you think we need to keep "zerion" in the config when we remove Valk?